identity/manager.go

```diff
@@ -62,6 +64,10 @@ func newManagerOptions(opts []ManagerOption) *managerOptions {
 }
 
 func (m *Manager) Create(ctx context.Context, i *Identity, opts ...ManagerOption) error {
+	if i.SchemaID == "" {
```

Comment:
Could you please provide a failing test case for the issue, which is then resolved by this change? That will help with any future regressions :)

Reply:
ping @splaunov

Reply:
Have added the test